### PR TITLE
Clean Managing Users section

### PR DESCRIPTION
**Pull request by nebelsprachen**
Renames files and ensure it meets mod docs criteria

### CODE REVIEW

File: server_admin/topics/users/con-user-impersonation.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// server_admin/topics/users.adoc
+
+[id="con-user-impersonation_{context}"]
+= User Impersonation
+
+An admin can impersonate a user.  For example, a user may be experiencing a bug in an applications and an admin can impersonate the user to investigate or duplicate the problem. Admins with the appropriate permission can impersonate a user.
```
Comment (andymunro) [inline server_admin/topics/users/con-user-impersonation.adoc]: change applications to singular and change admin to administrator.

Comment (andymunro) [reply server_admin/topics/users/con-user-impersonation.adoc]: Also, please change Admins to Administrators

File: server_admin/topics/users/con-user-impersonation.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+[id="con-user-impersonation_{context}"]
+= User Impersonation
+
+An admin can impersonate a user.  For example, a user may be experiencing a bug in an applications and an admin can impersonate the user to investigate or duplicate the problem. Admins with the appropriate permission can impersonate a user. 
+
+Any user with the `impersonation` role in the realm can impersonate a user. 
+
+image:{project_images}/user-details.png[]
+
+* If the admin and the user are in the same realm, then the admin will be logged out and automatically logged in as the user being impersonated.
```
Comment (andymunro) [inline server_admin/topics/users/con-user-impersonation.adoc]: administrator

File: server_admin/topics/users/con-user-impersonation.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+An admin can impersonate a user.  For example, a user may be experiencing a bug in an applications and an admin can impersonate the user to investigate or duplicate the problem. Admins with the appropriate permission can impersonate a user. 
+
+Any user with the `impersonation` role in the realm can impersonate a user. 
+
+image:{project_images}/user-details.png[]
+
+* If the admin and the user are in the same realm, then the admin will be logged out and automatically logged in as the user being impersonated.  
+* If the admin and user are not in the same realm, the admin will remain logged in, and additionally will be logged in as the user in that user's realm.  
+
+In both cases, the browser will be redirected to the impersonated user's *User Account Management* page.
```
Comment (andymunro) [inline server_admin/topics/users/con-user-impersonation.adoc]: administrator

File: server_admin/topics/users/con-user-impersonation.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Any user with the `impersonation` role in the realm can impersonate a user. 
+
+image:{project_images}/user-details.png[]
+
+* If the admin and the user are in the same realm, then the admin will be logged out and automatically logged in as the user being impersonated.  
+* If the admin and user are not in the same realm, the admin will remain logged in, and additionally will be logged in as the user in that user's realm.  
+
+In both cases, the browser will be redirected to the impersonated user's *User Account Management* page. 
+
+There are two locations an admin can access the *Impersonate* button to initiate user impersonation. You can access the *Impersonate* button from the *Details* tab or from the *Users* list.
```
Comment (andymunro) [inline server_admin/topics/users/con-user-impersonation.adoc]: I suggest: Two locations exists where an administrator can....

File: server_admin/topics/users/proc-configuring-user-attributes.adoc [hunk 5]
```diff
@@ -1,7 +1,11 @@
+// Module included in the following assemblies:
+//
+// server_admin/topics/users.adoc
+
 [id="proc-configuring-user-attributes_{context}"]
 = Configuring user attributes
 
-User attributes provide a customized experience for each user. With user attributes, you can can create a personalized identity for each user in the console.
+User attributes provide a customized experience for each user. You can can create a personalized identity for each user in the console by configuring user attributes.
```
Comment (andymunro) [inline server_admin/topics/users/proc-configuring-user-attributes.adoc]: duplicate "can"
Can can -- isn't that a dance?   :-)

File: server_admin/topics/users/proc-enabling-recaptcha-support.adoc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+. Select *Registration* from the drop down menu.
+. Set the *reCAPTCHA* requirement to *Required* by clicking the appropriate radio button. This enables
+reCAPTCHA.  
+. Click *Actions* to the right of the reCAPTCHA flow entry.
+. Click the *Config* link to redirect to the config page.
+. Enter the reCAPTCHA site key generated from the Google reCAPTCHA website, on the config page. 
+. Enter the secret generated from the Google reCAPTCHA website, on the config page.
+. Authorize Google to use the registration page as an iframe.
++
+NOTE: {project_name} prevents websites from including a login page dialog in an iframe. This is to prevent clickjacking attacks. You need to change the default HTTP response headers that is set in {project_name}.
```
Comment (andymunro) [inline server_admin/topics/users/proc-enabling-recaptcha-support.adoc]: Recommend always following "this" with a noun. This restriction ...

File: server_admin/topics/users/con-user-impersonation.adoc (hunk 3 above)
Comment (andymunro) [inline server_admin/topics/users/con-user-impersonation.adoc]: Change "will be redirected" to "redirects"

File: server_admin/topics/users/proc-enabling-terms-conditions.adoc [hunk 7]
```diff
@@ -1,12 +1,16 @@
...
+// con-required-actions.adoc
+
 [id="proc-enabling-terms-conditions_{context}"]
 = Enabling terms and conditions
 
-You can enable a required action that new users must accept the terms and conditions before logging into {project_name} for the first time.    
+You can enable a required action that new users must accept the terms and conditions before logging in to {project_name} for the first time.    
 
 .Procedure
 . Click the *Required Actions* tab.
```
Comment (andymunro) [inline server_admin/topics/users/proc-enabling-terms-conditions.adoc:11]: . Select a user.  (should be step 1)

File: server_admin/topics/users/ref-user-credentials.adoc [hunk 8]
```diff
@@ -12,7 +16,7 @@ Position::
    The arrow buttons in the *Position* column allow you to shift the priority of the credential for the user. The topmost credential has the highest priority. The priority determines which credential is displayed first after a user logs in.
 
 Type::
-   This shows the type of the credential, for example *password* or *OTP*.
+   This displays the type of the credential, for example *password* or *OTP*.
 
 User Label::
    This is an assignable label to recognize the credential when presented as a selection option during login. It can be set to any value to describe the
```
Comment (andymunro) [inline server_admin/topics/users/ref-user-credentials.adoc:22]: This column...

Comment (andymunro) [inline server_admin/topics/users/ref-user-credentials.adoc:22]: This column...

**Review by andymunro — CHANGES_REQUESTED** [review]
Excellent thorough work, @joaedwar.  It all looks so much better!!  Just a few minor improvements requested.

File: server_admin/topics/users/proc-configuring-user-attributes.adoc (hunk 5 above)
Comment (nebelsprachen) [reply server_admin/topics/users/proc-configuring-user-attributes.adoc]: haha :)

**Comment by nebelsprachen** [comment]
@andymunro  we need to standardize 'admin console'. 

Should it be::
-- Admin Console 
-- admin console
-- Admin console

I see instances of all of these in the doc set ?

**Review by andymunro — APPROVED** [review]

**Comment by andymunro** [comment]
Looks great, @joaedwar. Thanks for following up on everything as always very thoroughly.

**Pull request merged**